app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsCardItemFragment.kt

```diff
@@ -0,0 +1,439 @@
+                    ServiceFactory.get(WikiSite(Service.COMMONS_URL)).getImageInfo(pair.second, langFromCode)
+                            .subscribeOn(Schedulers.io())
+                            .observeOn(AndroidSchedulers.mainThread())
+                }
+                .subscribe({ response ->
+                    val page = response.query()!!.pages()!![0]
+                    if (page.imageInfo() != null) {
+                        val title = page.title()
+                        val imageInfo = page.imageInfo()!!
+
+                        sourceSummaryForEdit = PageSummaryForEdit(
+                                title,
+                                langFromCode,
+                                PageTitle(
+                                        Namespace.FILE.name,
+                                        StringUtil.removeNamespace(title),
+                                        null,
+                                        imageInfo.thumbUrl,
+                                        forLanguageCode(langFromCode)
+                                ),
+                                StringUtil.removeHTMLTags(title),
+                                fileCaption,
+                                imageInfo.thumbUrl,
+                                null,
+                                imageInfo.timestamp,
+                                imageInfo.user,
+                                imageInfo.metadata
+                        )
+
+                        targetSummaryForEdit = sourceSummaryForEdit!!.copy(
+                                description = null,
+                                lang = targetLanguage!!,
+                                pageTitle = PageTitle(
+                                        Namespace.FILE.name,
+                                        StringUtil.removeNamespace(title),
+                                        null,
+                                        imageInfo.thumbUrl,
+                                        forLanguageCode(targetLanguage!!)
+                                )
+                        )
+                    }
+                }, {
+                    L.e(it)
+                }))
+    }
+
+    private fun addImageTags() {
+        callToActionButton.text = context?.getString(R.string.suggested_edits_feed_card_add_image_tags)
+        disposables.add(EditingSuggestionsProvider
+                .getNextImageWithMissingTags(MAX_RETRY_LIMIT)
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe({ page ->
+                    imageTagPage = page
+                    updateUI()
+                }, {
+                    L.e(it)
+                }))
+    }
+
+    private fun showImageTagsUI() {
+        viewArticleImage.visibility = VISIBLE
+        viewArticleExtract.visibility = GONE
+        viewArticleTitle.visibility = GONE
+        showItemImage()
```

Comment:
By looking at the design, the image tags card should contain file name on the card.